rllib/evaluation/worker_set.py

```diff
@@ -638,16 +652,19 @@ def valid_module(class_path):
                 compress_columns=config["output_compress_columns"],
             )
 
-        # Assert everything is correct in "multiagent" config dict (if given).
-        ma_policies = config["multiagent"]["policies"]
-        if ma_policies:
-            for pid, policy_spec in ma_policies.copy().items():
+        if policy_specs:
+            policies = policy_specs
+        elif config["multiagent"]["policies"]:
+            # Make a copy so we don't modify the original multiagent config dict
+            # by accident.
+            ma_policies = config["multiagent"]["policies"].copy()
```

Comment:
Could we rename `ma_policies -> policy_specs` here for clarity? I guess it would be fine as in this case the incoming arg (`policy_specs`) is None anyways.

Reply:
ok. if we are up to renaming things, I think we should just name this policy_specs.

Reply:
yes, exactly.